rasa_launcher.py

```diff
@@ -0,0 +1,48 @@
+"""Makes all the necessary preparations to launch Rasa, and launches it.
+It relies on some linux-specific logic, and thus might not work in another OS.
+
+How to use:
+0. cd to the AI-replica dir
+1. python3 rasa_launcher.py
+2. After some time, the default browser will be opened, with the GUI to chat with the bot
+"""
+
+import sys
+import time
+
+from ai_replica.utils.system import (
+    create_env_if_not_exist,
+    install_piped_requirements,
+    execute_command,
+    open_url_in_browser, is_expected_python_version,
```

Comment:
Should the last import be placed on a new line?

Reply:
Fixed